### PR TITLE
[Bash/Sample] Bash script example for tflite image segmentation @open sesame 11/15 15:43

### PR DESCRIPTION
**Pull request by L2zz**
Related to #69

This PR contains two commits.
 1. Bash script example for tflite image segmentation
 -  It gets image from webcam and paints regions of objects detected. (* webcam is necessary)
 2. Meson build for tflite image segmentation example
 - Change related `meson build` for example

Signed-off-by: niklasjang niklasjang@gmail.com
Signed-off-by: Jihoon Lee ulla4571@gmail.com

### CODE REVIEW

File: bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+gst-launch-1.0 -v \
+      videomixer name=mix sink_0::alpha=0.7 sink_1::alpha=0.6 ! videoconvert !  videoscale ! autovideosink \
+      filesrc location=cat.png ! decodebin ! videoconvert ! videoscale ! imagefreeze !\
+      video/x-raw,format=RGB,width=257,height=257,framerate=10/1 ! tee name=t \
+      t. ! queue ! mix. \
+      t. ! queue ! tensor_converter !\
+      tensor_transform mode=arithmetic option=typecast:float32,add:0.0,div:255.0 !\
+      tensor_filter framework=tensorflow-lite model=tflite_img_segment_model/deeplabv3_257_mv_gpu.tflite !\
+      tensor_decoder mode=image_segment option1=tflite-deeplab ! mix.
```
Comment (jaeyun-jung) [inline bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: 1. tensor_transform option : 'add:0.0' is unnecessary
2. IMO, videoscale in the first line is unnecessary 
videomixer name=mix sink_0::alpha=0.7 sink_1::alpha=0.6 ! videoconvert ! autovideosink
3. can you test the camcoder? (v4l2src, I run below pipeline and work well)
```
gst-launch-1.0 -v \
videomixer name=mix sink_0::alpha=0.7 sink_1::alpha=0.6 ! videoconvert ! autovideosink \
v4l2src ! videoconvert ! videoscale ! \
video/x-raw,format=RGB,width=257,height=257,framerate=10/1 ! tee name=t \
t. ! queue ! mix. \
t. ! queue ! tensor_converter ! \
tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
tensor_filter framework=tensorflow-lite model=tflite_img_segment_model/deeplabv3_257_mv_gpu.tflite ! \
tensor_decoder mode=image_segment option1=tflite-deeplab ! mix.
```

Comment (L2zz) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: Thanks for comment! I will test code using camcoder and update to it.

Comment (wooksong) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: Q. Does tensor_decoder take float32 tensor as input? It seems that the data type of output tensor of the decoder should be uint8. Then, which element is in charge of typecast?

Comment (L2zz) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: Yes, tensor decoder gets float32 in range (0,1) which means probability that each pixel assumed to be some objects. In tensor decoder, it draws color map based on input. In other words, tensor decoder is in charge of typecast float32(probability) to uint32(rgba video).

Comment (myungjoo) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: @wooksong @jaeyun-jung PTAL?

Comment (jaeyun-jung) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: In my local, I couldn't run this pipeline (gstreamer pipeline grammar, need space)
Q) is it necessary to add decodebin and imagefreeze when running cam-src
```
gst-launch-1.0 -v \
      videomixer name=mix sink_0::alpha=0.7 sink_1::alpha=0.6 ! videoconvert ! autovideosink \
      v4l2src ! decodebin ! videoconvert ! videoscale ! imagefreeze ! \
      video/x-raw,format=RGB,width=257,height=257,framerate=10/1 ! tee name=t \
      t. ! queue ! mix. \
      t. ! queue ! tensor_converter ! \
      tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
      tensor_filter framework=tensorflow-lite model=tflite_img_segment_model/deeplabv3_257_mv_gpu.tflite ! \
      tensor_decoder mode=image_segment option1=tflite-deeplab ! mix.
```

Comment (L2zz) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: Sorry imagefreeze is for previous example which uses image. 
Decodebin is for webcam compatibility related with issue #61. 
In my case, my webcam returns jpeg/image data which cause internal data stream error.
So I put decodebin to make video/x-raw from jpeg.

Comment (jaeyun-jung) [reply bash_script/example_image_segmentation_tensorflow_lite/gst-launch-image-segmentation-tflite.sh]: Thank you for your updation, now CI has some error and never mind the build error.

**Pull request merged**